frontends/web/src/api/bitbox02.ts

```diff
@@ -81,3 +81,9 @@ export const setMnemonicPassphraseEnabled = (
       return Promise.resolve();
     });
 };
+
+export const verifyAttestation = (
+  deviceID: string,
+): Promise<boolean> => {
```

Comment:
Fyi the return type of this API call is `null | boolean`, see here:

https://github.com/digitalbitbox/bitbox-wallet-app/blob/e2f1008475cad14c7ecf9f73011f819a18d03e6a/backend/devices/bitbox02/handlers/handlers.go#L128

https://github.com/digitalbitbox/bitbox02-api-go/blob/e9258db31750ed3ba2b4e988b4bd74c9c4107f64/api/firmware/attestation.go#L166-L167

![image](https://user-images.githubusercontent.com/1225267/193294121-878a5e0d-5742-4c68-a8c1-609053ef4725.png)

cc @Beerosagos @thisconnect

Reply:
Was also typed like this already here: https://github.com/digitalbitbox/bitbox-wallet-app/pull/1773/files#diff-ddc6137588aa6e257eb264fca22e2ab9ab7d8183da99c07fe4df15b58c2e0ab3R52